gui/src/2D/controls/image.ts

```diff
@@ -260,6 +260,28 @@ export class Image extends Control {
         this._markAsDirty();
     }
 
+    /** @hidden */
+    public _rotate90(n: number): Image {
+        let canvas = document.createElement('canvas');
+
+        const context = canvas.getContext('2d')!;
+        const width = this._domImage.width;
+        const height = this._domImage.height;
+
+        canvas.width = height;
+        canvas.height = width;
+
+        context.translate(canvas.width / 2, canvas.height / 2);
+        context.rotate(n * Math.PI / 2);

```

Comment:
if n == 2 then width and height should not be switched...It works only if n is odd

Reply:
Only values used are -1 and 1. The function is only intended to be used internally.

Reply:
Ok makes sense